migrations.lock

```diff
@@ -921,7 +923,6 @@ two_factor
  0004_auto_20160205_1827
  0005_auto_20160224_0450
  0006_phonedevice_key_default
- 0007_auto_20201201_1019
```

Comment:
is the deletion of this migration intentional? did you `pip install` or `pip sync` requirements lately? this doesn't look related to the oauth work

Reply:
I didn't delete this intentionally. I have done `pip install requirements` today